src/lib/OTA/OTA.cpp

```diff
@@ -106,7 +108,13 @@ static void ICACHE_RAM_ATTR PackChannelDataHybridCommon(OTA_Packet4_s * const ot
     // CRSF input is 11bit and OTA will carry only 10bit. Discard the Extended Limits (E.Limits)
     // range and use the full 10bits to carry only 998us - 2012us
     PackUInt11ToChannels4x10(&channelData[0], &ota4->rc.ch, &Decimate11to10_Limit);
-    ota4->rc.ch4 = CRSF_to_BIT(channelData[4]);
+
+    // send armed status to receiver
+    #if defined(UNIT_TEST)
+    ota4->rc.isArmed = CRSF_to_BIT(channelData[4]);
+    #else
+    ota4->rc.isArmed = handset->IsArmed();
+    #endif
```

Comment:
I don't really like this.
The unit tests should probably have a mock handset laying around for this to work correctly.

Reply:
We can do this part later though.

Reply:
Yes, please.